pkg/services/sqlstore/alert.go

```diff
@@ -201,6 +205,22 @@ func updateAlerts(existingAlerts []*m.Alert, cmd *m.SaveAlertsCommand, sess *DBS
 
 			sqlog.Debug("Alert inserted", "name", alert.Name, "id", alert.Id)
 		}
+
+		tags := alert.GetTagsFromSettings()
```

Comment:
I would like to see some kind of simple test for this. This part of the codebase is not tested enough.

Reply:
Done